{{cookiecutter.project_name}}/pyproject.toml

```diff
@@ -44,6 +44,8 @@ safety = "^3.1.0"
 pytest-randomly = "^3.15.0"
 faker = "^28.4.1"
 factory-boy = "^3.3.1"
+deptry = "0.21.1"
+fawltydeps = "0.17.0"
```

Comment:
if you don't mind, remove it and commented line for `fawltydeps` in the `Makefile`.